update.json

```diff
@@ -0,0 +1,12 @@
+{
+  "addons": {
+    "remote-settings-devtools@mozilla.com": {
+      "updates": [
+        {
+          "version": "1.3.0",
+          "update_link": "https://github.com/mozilla-extensions/remote-settings-devtools/releases/download/1.3.0/remote-settings-devtools-1.3.0.xpi"
```

Comment:
Just to note this doesn't work at the moment, as the name of the xpi is different. However, I think if you start uploading direct from the builders, then you'll get the version number there by default, so it probably won't be an issue.

Reply:
I don't really how to upload direct from the builders :/ 

At least, I'd like to a proper release procedure with clear steps in the README. I will update the PR based on your comments

Reply:
Oh sorry that was confusing. I meant that you download it, and then upload it without renaming (I think the searchengine-devtools ones have the version in automatically).

And yes, I should document the process.